src/Insights/Customizations/Shoebox/FilterDimension.cs

```diff
@@ -0,0 +1,78 @@
+﻿//------------------------------------------------------------
```

Comment:
This copyright header for this file and other files - not even in the PR - is inconsistent with one used among all other projects. Please update all of them